Xamarin.Essentials/Permissions/Permissions.android.cs

```diff
@@ -41,8 +41,6 @@ static void PlatformEnsureDeclared(PermissionType permission)
 
         static Task<PermissionStatus> PlatformCheckStatusAsync(PermissionType permission)
         {
-            EnsureDeclared(permission);
```

Comment:
I think this needs to stay in.  This is simply checking the manifest to ensure it's declared and should not require UI thread for _this_ portion.